jetty-io/src/main/java/org/eclipse/jetty/io/SelectorManager.java

```diff
@@ -410,8 +408,6 @@ public boolean addEventListener(EventListener listener)
     @Override
     public boolean removeEventListener(EventListener listener)
     {
-        if (isRunning())
```

Comment:
I think this definitely exposes us to `ConcurrentModificationException`s if the selector is iterating over events when this is called.